### ISSUE TITLE
Clarify sentence beginning with "In     support of the particular set of problems described here...."

### ISSUE THREAD

**Issue by rdroms** [Abhayakara/draft-tldr-sutld-ps#17]
From Suzanne Woolf
https://www.ietf.org/mail-archive/web/dnsop/current/msg19170.html

Introduction

4. STYLE/GRAMMAR: This sentence is hard to parse: "In 
   support of the particular set of problems described here...."

OLD:
   "In support of the particular set of problems described here, the
   document also includes documentation of existing practice as it
   relates to the use of Domain Names, as well as a brief history of
   Domain Names, and finally to describe the set of problems that exist
   as reported by various IETF participants with experience in the
   various aspects of the problem."

NEW:
"In support of its analysis of the particular set of issues described here, the document also includes descriptions of existing practice as it relates to the use of domain names, a brief history of domain names, and some observations by various IETF participants who have experience with various aspects of the current situation."

**Comment by wkumari** [comment]
Thank you. Seems much more readable.

W
